suite-native/notifications/src/components/TransactionNotification.tsx

```diff
@@ -1,92 +1,122 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useCallback } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 
 import { useNavigation } from '@react-navigation/native';
 
-import { AccountKey, TransactionType } from '@suite-common/wallet-types';
+import { TransactionType } from '@suite-common/wallet-types';
 import { TransactionIcon } from '@suite-native/transactions';
 import {
     TransactionsRootState,
-    selectTransactionByTxidAndAccountKey,
     selectTransactionFirstTargetAddress,
+    AccountsRootState,
+    selectAccountKeyByDescriptorAndNetworkSymbol,
 } from '@suite-common/wallet-core';
 import {
     RootStackRoutes,
     StackNavigationProps,
     RootStackParamList,
 } from '@suite-native/navigation';
 import { Icon } from '@trezor/icons';
+import {
+    EventNotification,
+    notificationsActions,
+    TransactionEventNotification,
+} from '@suite-common/toast-notifications';
 
 import { Notification } from './Notification';
 import { TransactionNotificationDescription } from './TransactionNotificationDescription';
 
 type TransactionNotificationProps = {
-    txid: string;
-    accountKey: AccountKey;
+    notification: TransactionEventNotification;
```

Comment:
nit: better name would be transactionNotification because it's weird when down below you access notification txid. I would like to have the transaction in naming.

Reply:
is not relevant after refactor i 53e695678